apps/server/src/shared/domain/domainobject/board/column-board.do.ts

```diff
@@ -13,4 +13,12 @@ export class ColumnBoard extends BoardComposite implements BoardNodeBuildable {
 	useBoardNodeBuilder(builder: BoardNodeBuilder, parent?: AnyBoardDo): void {
 		builder.buildColumnBoardNode(this, parent);
 	}
+
+	accept(visitor: BoardCompositeVisitor): void {
```

Comment:
If the code strucure following a composit pattern, "i think" it should not nesassary that accept and acceptAsync must be added in each of the implementations. I am not sure if the code work only with interfaces for now, or if you also have a abstract class that can handle accept and acceptAsync on one place for you.

The visitor know what they should be do, but the acceptor do not know anything about the visitor. 
To the acceptor it can be pushed different visitors with different think that they should be do. 

visitor.visitColumnBoardAsync include specification in name. Yes it is matched to the colum. But i think BoardCompositeVisitorAsync is not the visitor, it is a super class that collected different visitors for now and "visitColumnBoardAsync" is the real visitor. 
visitor.visit(this) is what i think should be executed on this place and then it is the same for all childs of the composit and can be defined one time. 

This also effects the super class structure. 

I must google the differents between listener, observer and visitor it mixed a litte bit in my mind. :D
https://kb.novaordis.com/index.php/Differences_between_Observer,_Listener_and_Visitor_Patterns